src/style.css

```diff
@@ -0,0 +1,30 @@
+@import url(/src/styles.css/stars.css);
+@import url(/src/styles.css/card.css);
+
+@font-face {
+  font-family: roboto;
+  src: url(/src/fonts/Roboto/Roboto-Regular.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: roboto;
+  src: url(/src/fonts/Roboto/Roboto-Bold.ttf);
+  font-weight: 700;
+}
+
+@font-face {
+  font-family: roboto;
+  src: url(/src/fonts/Roboto/Roboto-Medium.ttf);
+  font-weight: 500;
```

Comment:
I think this is the default value, so you can leave it blank

Reply:
font-weight: 400; - is default